support/network_of_terms_acl.erl

```diff
@@ -0,0 +1,31 @@
+%% @doc ACL checks for usage of this mod.
+-module(network_of_terms_acl).
+-author("Driebit <info@driebit.nl>").
+
+-export([
+    is_allowed/1
+]).
+
+-include("zotonic.hrl").
+
+
+%% @doc Check whether the user is allowed to use this mod or not.
+%% Note that by default all users are allowed, but one can forbid the 'use'
+%% of 'mod_network_of_terms' to restrict access to it (except for the admin).
+%%
+%% For debugging purposes, a debug-level message is logged when access is non-default.
+is_allowed(Context) ->
+    case z_acl:is_admin(Context) of
+        true -> true;
+        _ -> case z_acl:is_allowed(use, mod_network_of_terms, Context) of
+            undefined -> true;
+            Result ->
+                z:debug(
+                    "Non-default access to mod_network_of_terms for user ~p. Allowed: ~s",
+                    [ Context#context.user_id , Result ],
+                    [],
+                    Context
+                ),
+                Result
+        end
+    end.
```

Comment:
Good one to use the tri-state nature of `is_allowed` (true/allowed, false/disallowed and undefined/not set); defaulting to true on undefined makes it backwards compatible.
I'm on the fence on whether the debug log needs to stay after we tested it - the allow logic is quite transparent, because it strictly follows the normal module use rules, inspectable in the ACL configuration admin. Ah, unless it is set by notification, hmm ok so maybe indeed nice to keep it.